src/components/Layout.tsx

```diff
@@ -51,6 +51,9 @@ const Layout: React.FC<IProps> = ({
   useEffect(() => {
     if (path.includes("/docs/")) {
       setShouldShowSideNav(true)
+    } else {
+      !path.includes("/docs/")
```

Comment:
What is this line for?
```suggestion
```

Reply:
Yeah looks like this may have been left over as a note, but isn't needed for the logic since the `else` statements covers it already.

Reply:
Correct. Ok, I'll remove it for now.